### PR TITLE
Update Documentation in User's Guide

### PR DESCRIPTION
**Pull request by marcelofcastro**
This PR tries to include every piece of documentation we have to be inside the library, within user's guide.

### CODE REVIEW

**Comment by dietmarw** [comment]
I've added some clean up. Mind only your table should be put under "revisions" any other information should stay in "info". There are also still occurrences of email addresses for contact that need to be updated.

**Comment by dietmarw** [comment]
One thing that is missing is that we should probably make use of the domain openipsl.org also in the User's Guide and top level package documentation.

**Comment by dietmarw** [comment]
Would it be a good idea to sort the publications by year and replace all those "links" by doi links?

**Comment by marcelofcastro** [comment]
> I've added some clean up. Mind only your table should be put under "revisions" any other information should stay in "info". There are also still occurrences of email addresses for contact that need to be updated.

Yeah, I know that. I might have forgot about one or two. I will check that now.

**Comment by marcelofcastro** [comment]
> Would it be a good idea to sort the publications by year and replace all those "links" by doi links?

maybe when we have the entire list updated for 2.0

File: OpenIPSL/UsersGuide/package.mo [hunk 1]
```diff
@@ -9,6 +9,7 @@ The <b>Open-Instance Power System Library - OpenIPSL</b> contains many power sys
 </p>
 <p>
 This is a very short <b>User's Guide</b> that will try to help users to get familiar with the library providing general information about the <b>OpenIPSL</b>.
+More information about the library can be found in this <a href=\"https://openipsl.readthedocs.io/en/latest/\">website</a> dedicated to the OpenIPSL organization. 
 </p>
```
Comment (dietmarw) [inline OpenIPSL/UsersGuide/package.mo:13]: I think you got the wrong address here. readthedocs should not appear anywhere. The content should actually be removed. It is openipsl.org that is the correct address.

File: OpenIPSL/package.mo [hunk 2]
```diff
@@ -25,6 +25,8 @@ If you are looking for an introduction to the library, have especially a look at
 <ul>
 <li><a href=\"modelica://OpenIPSL.UsersGuide\">User's Guide</a>
   and all its dependencies for general information about the library.</li>
+<li><a href=\"https://openipsl.readthedocs.io/en/latest/\">openipsl.org</a>,
```
Comment (dietmarw) [inline OpenIPSL/package.mo:28]: same problem here.

File: OpenIPSL/UsersGuide/package.mo [hunk 3]
```diff
@@ -9,6 +9,7 @@ The <b>Open-Instance Power System Library - OpenIPSL</b> contains many power sys
...
 <p>
 This is a very short <b>User's Guide</b> that will try to help users to get familiar with the library providing general information about the <b>OpenIPSL</b>.
+More information about the library can be found in this <a href=\"https://openipsl.readthedocs.io/en/latest/\">website</a> dedicated to the OpenIPSL organization.
```
Comment (marcelofcastro) [inline OpenIPSL/UsersGuide/package.mo:12]:
```suggestion
More information about the library can be found in this <a href=\"https://openipsl.org\">website</a> dedicated to the OpenIPSL organization. 
```

File: OpenIPSL/UsersGuide/package.mo (hunk 1 above)
Comment (marcelofcastro) [reply OpenIPSL/UsersGuide/package.mo:13]: is this what you meant?

File: OpenIPSL/package.mo (hunk 2 above)
Comment (marcelofcastro) [inline OpenIPSL/package.mo:28]:
```suggestion
<li><a href=\"https://openipsl.org\">openipsl.org</a>, 
```

**Review by dietmarw — APPROVED** [review]
There are still some formulations in the User's Guide that I would like to go through but that can wait for the final release. Good enough for now.

**Review by dietmarw — CHANGES_REQUESTED** [review]
I forgot one thing. This pull request should also remove all the now superfluous sphinx documentation files that are still present.

**Comment by marcelofcastro** [comment]
@dietmarw do you want to keep something or do you just want to remove the docs folder (I don't remember if that's the folder that only contains the website stuff)?

**Review by dietmarw — APPROVED** [review]
Since the PR needed to get rebased on latest branch any way I've now also removed the `doc/` directory at the same time. It's good to merge now.

**Review by dietmarw — CHANGES_REQUESTED** [review]
Sorry for going back and forth on this one. But I remembered that we are still missing the links to the verification data that is hosted on the dedicated repository. So that information should added to the verification section.

**Comment by marcelofcastro** [comment]
The repository itself is private right now. When we make it public we can put the link to the repo. However, we already have the link to the website with the comparison results.

**Comment by dietmarw** [comment]
Yes I was only talking about the website, the the source repository

**Comment by marcelofcastro** [comment]
But I have already mentioned the website in the result verification page.

File: OpenIPSL/UsersGuide/LibraryTesting.mo [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+<h4>Regression Testing</h4>
+<p>These tests are used to ensure the validation status of models that are based on PSSE implementations. 
+The models undergo simulations with different events such as faults, reference changes and load variations.
+The main idea is to try to capture different responses from the model being verified. If a model has a very small error if compared to PSSE base result for that simulation, it passes a test.
+A model is considered to be verified if it passes all tests. Models that pass the verification procedure have a green dashed line around them.</p>
+
+<p>The regression testing is done separately in different Modelica-compliant tools (<a href=\"https://www.openmodelica.org/\">OpenModelica</a>  and <a href=\"http://www.3ds.com/products-services/catia/products/dymola\">Dymola</a>, for now) so models can be independently verified.
+A detailed view of all models that undergo the verification process can be found 
+in the <a href=\"https://alsetlab.github.io/NYPAModelTransformation/reports\">NYPA Model Transformation
```
Comment (marcelofcastro) [inline OpenIPSL/UsersGuide/LibraryTesting.mo:19]: @dietmarw Maybe we can just make explicit that the results are there, right?

Comment (dietmarw) [reply OpenIPSL/UsersGuide/LibraryTesting.mo:19]: Ah yes of course. In that case we are good.

**Review by dietmarw — APPROVED** [review]

**Pull request merged**